FileIO/CsvInterface.cpp

```diff
@@ -74,12 +74,13 @@ int CsvInterface::readPoints(std::string const& fname, char delim,
 
 	std::string line;
 	getline(in, line);
-	std::array<std::size_t, 3> const column_idx = 
-		{ CsvInterface::findColumn(line, delim, x_column_name),
-		  CsvInterface::findColumn(line, delim, y_column_name),
-		  (z_column_name.empty()) ? CsvInterface::findColumn(line, delim, y_column_name) : 
-		                            CsvInterface::findColumn(line, delim, z_column_name) };
-	
+	std::array<std::size_t, 3> const column_idx =
+		{{ CsvInterface::findColumn(line, delim, x_column_name),
+			CsvInterface::findColumn(line, delim, y_column_name),
```

Comment:
A discussion with Lars hopefully enlighted me. There is a difference between indentation and alignment. Imho indentation only is better than indentation + alignment. But here the opinions are different. I will revert changes above.